grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -1381,6 +1647,7 @@ def section_streaming_actors(outer_panels: Panels):
                         ),
                     ],
                 ),
+                panels.subheader("Over Window"),
```

Comment:
I think we can move OverWindow down below `TopN`, cuz it's less frequently used than `Join` and `Aggregate`, and is logically related to `TopN`.